src/test/java/org/openmrs/kenyaemr/contrib/qaframework/automation/page/LoginPage.java

```diff
@@ -10,6 +10,7 @@
 package org.openmrs.kenyaemr.contrib.qaframework.automation.page;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.By.ByCssSelector;
```

Comment:
you don't need to import individual package for each selenium locator(id, cssSelector, className, etc) since
```
import org.openqa.selenium.By;
```
contains all locators.
```suggestion
import org.openqa.selenium.By.ByCssSelector;
```